addons/binding/org.openhab.binding.foobot/src/main/java/org/openhab/binding/foobot/internal/FoobotConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.foobot.internal;
+
+/**
+ * The {@link FoobotConfiguration} class contains fields mapping thing configuration paramters.
+ *
+ * @author Divya Chauhan - Initial contribution
+ */
+public class FoobotConfiguration {
+
+    public String apikey;
+
+    public String username;
+
+    public String mac;
+
+    public Integer refresh;
+
+    public String uuid;
```

Comment:
It doesn't look to me like configuring the `uuid` directly is supported, so this field can be omitted.